game/scripts/npc/items/custom/item_cave_rune_haste_doubledamage.txt

```diff
@@ -0,0 +1,38 @@
+"DOTAItems"
+{
+//============================================================================================================

```

Comment:
Why?

Reply:
It's in every other file of the type and looks pretty

Reply:
So pretty we need to add 2 sections of them? :D